bi-connectors/TableauConnector/src/manifest.xml

```diff
@@ -2,8 +2,8 @@
 
 <connector-plugin class='opensearch_jdbc' superclass='jdbc' plugin-version='2.0.0.0' name='OpenSearch' version='18.1' min-version-tableau='2021.1'>
```

Comment:
Should we update the version? Is it JDBC driver version or connector version?

Reply:
I'm not sure. Or is this the SQL plugin version? We never had a 2.x JDBC version